libfreerdp/core/freerdp.c

```diff
@@ -562,8 +562,7 @@ const char* freerdp_get_version_string(void)
 
 const char* freerdp_get_build_date(void)
 {
-	static char build_date[] = __DATE__ " " __TIME__;
```

Comment:
Note that gcc already has patches to override these from SOURCE_DATE_EPOCH.
upstream in gcc7 and backported to gcc6 in Debian and openSUSE, so this part is not really needed